developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -400,7 +400,158 @@ response.data?.Get?.Article?.[0]['_additional']?.creationTimeUnix // Get the tim
 
 ## How to migrate your code
 
-For code examples, see the pages here:
+
+This workflow describes the process one would follow to migrate their codebase to the v3 client. 
+
+1. Install the v3 client package. Update the client package from the v2 to v3 client. 
+
+2. Connect to Weaviate. Make sure your client instantiation respects the new clients syntax. Our documentation on instantiation covers use cases from local, custom to WCS.
```

Comment:
Consider:

```
1. Connect to Weaviate. Make sure your client instantiation uses the v3 connection syntax. For examples, see [How-to connect](link).
```

Question: 
Assuming they have different names, could a user app run with two client connections, one v2 and one v3? Because if the client connection changes, I think that would break all of the v2 code.

The Py4 client can run Py3 code, but only if the client object is instantiated using the v3 style.